pyupgrade.py

```diff
@@ -458,6 +524,8 @@ def _imports_unicode_literals(contents_text):
         else:
             return False
 
+    return False
```

Comment:
mypy caught this

Reply:
not sure this is possible 🤷‍♂

Reply:
tell it to mypy? :/

Reply:
oh it definitely is possible, heh -- the testsuite even exercises it!

Reply:
I guess the test suite doesn't depend on this being a true bool then